homeassistant/components/rpi_power/binary_sensor.py

```diff
@@ -31,43 +34,22 @@ async def async_setup_entry(
 class RaspberryChargerBinarySensor(BinarySensorEntity):
     """Binary sensor representing the rpi power status."""
 
-    def __init__(self, under_voltage):
+    def __init__(self, under_voltage: UnderVoltage) -> None:
         """Initialize the binary sensor."""
         self._under_voltage = under_voltage
-        self._is_on = None
+        self._attr_is_on = None
+        self._attr_device_class = DEVICE_CLASS_PROBLEM
+        self._attr_icon = "mdi:raspberry-pi"
+        self._attr_name = "RPi Power status"
+        self._attr_unique_id = "rpi_power"  # only one sensor possible
```

Comment:
These are all fixes/static and can be set as a class attribute instead of a property.

Reply:
Almost all places I found are using `__init__` to set static attributes. judging by your comments, that should be changed (updated).
@cdce8p can pylint check this?

Reply:
Its not required. Its a common practice we apply. We don't need to check for this, nor do you have to ping/mention people for this.